Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -310,7 +314,7 @@ def IccRightChart (x y : ℝ) [h : Fact (x < y)] :
 /-- Charted space structure on `[x, y]`, using only two charts taking values in
 `EuclideanHalfSpace 1`.
 -/
-instance IccManifold (x y : ℝ) [h : Fact (x < y)] :
```

Comment:
I agree that renaming this to `IccChartedSpace` makes sense. I have used this instance name, though --- so I'm not sure if removing it actually improves anything. (What does the autogenerated name look like?)

Reply:
`IccChartedSpace` sounds good. The auto generated names are ugly. See the [two instances that follow](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Geometry/Manifold/Instances/Real.html#instChartedSpaceEuclideanHalfSpaceOfNatNatElemRealIcc). Should we rename those two as well? Perhaps `IccZeroOneChartedSpace`? I don't see these names used anywhere within mathlib, so I assume your usage is confined to a branch of yours.

Reply:
Renaming those seems fine. I would leave out the 0 and 1 from the names, though --- mathematically, these are not important (to me). My usage is in a PR to mathlib, and indeed not merged yet.

Reply:
On 0 and 1, see my comment below.